src/components/ReportActionItem/TransactionPreview/TransactionPreviewContent.tsx

```diff
@@ -142,23 +139,36 @@ function TransactionPreviewContent({
         sortedParticipantAvatars.push(getWorkspaceIcon(chatReport));
     }
 
+    // Compute the from/to data only for IOU reports
+    const {from, to} = useMemo(() => {
+        if (!shouldShowPayerAndReceiver) {
+            return {
+                from: undefined,
+                to: undefined,
+            };
+        }
+
+        // For IOU, we want the unprocessed amount, because it is important whether the amount was positive or negative
+        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+        const transactionRawAmount = (transaction?.modifiedAmount || transaction?.amount) ?? 0;
```

Comment:
this is the actual heart of the fix - it will pass the unmodified amount, so we will get the positive or negative number and can properly render the from/to arrow